### ISSUE TITLE
kaseya web app. 

### ISSUE THREAD

**Issue by jwoodard80** [wavebox/waveboxapp#69]
Love this so far, however, I'm wondering if you could look into support for kaseya web app? I can understand if this isn't possible but I have all my webapps in one window now and am thrilled. I am just having to open a seperate browser for kaseya and was hoping something could be done to fix this. :-)

thanks.

* Which version of Wavebox are you using? [e.g. 3.0.0]
Wavebox 3.1.3

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Windows 10

**Comment by Thomas101** [comment]
We'll take a look, see how this can work! You might be able to just add it with the `Add Any Website` account. It's not got the tight integration the other account types do, but might work as a viewer

**Comment by Dark-Witcher** [comment]
Thing is, I can't. Right now the only options that I can access is switch between my 2 gmail accounts from the system tray icon. I will probably have to reinstall the app to fix this, but I think that there should be a settings option from right-click menu of the system tray icon.

**Comment by jwoodard80** [comment]
Agreed, that is actually what I was trying. I was able to get a couple of other sites working GREAT! And actually I was able to get kaseya working. I think it has something to do with re-direction. We logged in at kaseya.ourdomain.com and we are re-directed to something else xyz.kaseya.com. I put in the xyz.kaseya.com and it worked. 

Working on others (ubiquiti unifi is giving me fits) but I'm hoping to make progress! love it so far and look to be a customer soon!
